From 258c2b39f4c07cfc8f9c046f30b956d4d2a17b1d Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 11 May 2017 15:57:06 +0300
Subject: [PATCH 1789/5242] MLK-14800: fbdev: Refactor printed information

commit  7071b3aeaf15cca4f9a532c579ac0083ecbd5e3f from
https://source.codeaurora.org/external/imx/linux-imx.git

Debug info should not be printed using pr_info on init. Use pr_debug
instead.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/adv7535.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/adv7535.c b/drivers/video/fbdev/mxc/adv7535.c
index 27c99a2..3035dc5 100644
--- a/drivers/video/fbdev/mxc/adv7535.c
+++ b/drivers/video/fbdev/mxc/adv7535.c
@@ -326,13 +326,13 @@ static __init int adv7535_init(void)
 {
 	u8 err = 0;
 
-	pr_info("In adv7535_init\n");
 
 	err = i2c_add_driver(&adv7535_i2c_driver);
 	if (err != 0)
 		pr_err("%s: i2c driver register failed, error = %d\n",
 			__func__, err);
 
+	pr_debug("%s (ret=%d)\n", __func__, err);
 	return err;
 }
 
-- 
1.7.9.5

